spec/supabase_js_v2.yml

```diff
@@ -480,7 +490,8 @@ functions:
           ```
       - id: update-password-with-reauthentication
         name: Update the user's password with a nonce
-        description: If "Secure password change" is enabled, updating the user's password would require a nonce. The nonce is sent to the user's email or phone number.
+        description: |
+          If **Secure password change** is enabled in your [project's email provider settings](/project/_/auth/providers), updating the user's password would require a nonce if the user **hasn't recently signed in**. The nonce is sent to the user's email or phone number. A user is deemed recently signed in if the session was created in the last 24 hours.
```

Comment:
should this be `project/_/auth/providers`? Perhaps `https://app.supabase.com/project/_/auth/providers`?

Reply:
relative links would work here since the platform can be accessed from the `https://supabase.com` domain too but i just realised that it should be `/dashboard/project/_/auth/providers` - will make the change